supergloo/docs/mesh/mesh-discovery.md

```diff
@@ -0,0 +1,146 @@
+istio-system       istio-citadel-6559b9697b-wgcqm            1/1     Running     0          88s
+istio-system       istio-cleanup-secrets-lw9xm               0/1     Completed   0          89s
+istio-system       istio-egressgateway-5dd6dbb89f-pb8bn      1/1     Running     0          89s
+istio-system       istio-galley-7cd7dc49f9-m95ws             1/1     Running     0          89s
+istio-system       istio-ingressgateway-947b9cd66-bq5fs      1/1     Running     0          89s
+istio-system       istio-pilot-7786dc9c69-p7cjn              2/2     Running     0          88s
+istio-system       istio-policy-6c54647778-zmtb8             2/2     Running     0          88s
+istio-system       istio-security-post-install-rlw9d         0/1     Completed   0          89s
+istio-system       istio-sidecar-injector-75fddbd5c9-97rs2   1/1     Running     0          88s
+istio-system       istio-telemetry-68c686dd4b-4q9ml          2/2     Running     0          88s
+istio-system       prometheus-578b7dcfdc-gnx2x               1/1     Running     0          88s
+kube-system        coredns-fb8b8dccf-dl2bj                   1/1     Running     0          28h
+kube-system        coredns-fb8b8dccf-x966j                   1/1     Running     0          28h
+kube-system        etcd-minikube                             1/1     Running     0          28h
+kube-system        kube-addon-manager-minikube               1/1     Running     0          28h
+kube-system        kube-apiserver-minikube                   1/1     Running     0          28h
+kube-system        kube-controller-manager-minikube          1/1     Running     0          28h
+kube-system        kube-proxy-cdmpd                          1/1     Running     0          28h
+kube-system        kube-scheduler-minikube                   1/1     Running     0          28h
+kube-system        storage-provisioner                       1/1     Running     0          28h
+supergloo-system   discovery-58d85b9d9f-nkkpk                1/1     Running     0          9s
+supergloo-system   mesh-discovery-7b69d48d7c-f8cmt           1/1     Running     0          9s
+supergloo-system   supergloo-84f85b459c-sfvdg                1/1     Running     0          9s
+```
+
+#### Discovered Mesh CRD
+
+Once istio and supergloo are running in the cluster we can check for the mesh CRD:
+```bash 
+➜ kubectl get mesh -n supergloo-system -oyaml
+apiVersion: supergloo.solo.io/v1
+kind: Mesh
+metadata:
+  generation: 2
+  name: istio-istio-system
+  namespace: supergloo-systeme
+spec:
+  discoveryMetadata:
+    injectedNamespaceLabel: istio-injection
+    installationNamespace: istio-system
+    meshVersion: 1.0.6
+  istio:
+    installationNamespace: istio-system
+    istioVersion: 1.0.6
+status:
+  reported_by: istio-config-reporter
+  state: 1
+```
+
+As you can see supergloo figured out the localtion and version of istio, and now we can go ahead and apply supergloo rules to our mesh.
```

Comment:
Is that all it does? Can we be more forward looking with what we expect it to do and the overall power this discovery process will bring to SuperGloo (and possibly other projects)?